### PR TITLE
Added scripts for base-image_creation and cleanup tasks

### PR DESCRIPTION
**Pull request by kbidarkar**
a) base-image_creation scripts help create base_images for rhel6
   or rhel7.
b) Helps in cleaning up stray vms and docker containers, this would
   be helpful if teardown misses cleanup and could be added as
   cronjobs.

### CODE REVIEW

**Comment by elyezer** [comment]
ACK, pending a nitpick: `s/base-image_creation/base_image_creation

File: misc/base-image_creation/create-base-image.sh [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    --ram 4096 \
+    --vcpus=2 \
+    --check-cpu \
+    --accelerate \
+    --hvm \
+    --location=$os_url \
+    --cpu host \
+    --force
+
+# The above --cpu host is required if the need is to setup a vm for sat6 with provisioning support. What this does is it enables nested virtualization.
```
Comment (sthirugn) [inline misc/base-image_creation/create-base-image.sh]: How about 
```sh
# The argument `--cpu host` enables nested virtualization and is required to setup sat6 vms with provisioning support.
```

**Comment by sthirugn** [comment]
ACK pending non-blocker comment

**Comment by kbidarkar** [comment]
Using this same code pushed, I have now created a base-image for rhel72 using the latest-build.
This image also points to the internal rhel72 repo.

**Comment by elyezer** [comment]
ACK, will let @sthirugn take another look before merging.

**Comment by sthirugn** [comment]
:+1:

**Comment by sghai** [comment]
perfect.. thanks @kbidarkar for adding these scripts here.

**Pull request merged**